docs/docs/personalization.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Personalization
+---
+
+## What is Personalization?
+
+- Have you ever landed on a website that greeted you by name?
+
+- Did you ever search for a book and realize the suggested results were based on your previous purchases?
+
+These are examples of personalization. Personalization adjusts the content of a website, and creates a custom experience for users. With personalization, one website can offer a variety of experiences to different visitors.
+
+## Why is Personalization Important?
```

Comment:
Updating to follow the Gatsby Style Guide for sentence case: [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/)
```suggestion
## Why is personalization important?
```